external/common_functions.sh

```diff
@@ -32,214 +32,190 @@ supported_builds="full"
 # Supported tests
 supported_tests="external_custom camel criu-portable-checkpoint criu-portable-restore criu-ubi-portable-checkpoint criu-ubi-portable-restore derby elasticsearch jacoco jenkins functional-test kafka lucene-solr openliberty-mp-tck payara-mp-tck quarkus quarkus_quickstarts scala system-test tomcat tomee wildfly wycheproof netty spring zookeeper"
 
-function check_version() {
-    version=$1
-
-    for current_version in ${supported_versions}
-    do
-        if [[ "${version}" == "${current_version}" ]]; then
-            return 1
-        fi
-    done
-
-    return 0
-}
-
-# Set a valid version
-function set_version() {
-    version=$1
-
-    if (check_version ${version}); then
-        echo "ERROR: Invalid Version: ${version}"
-        echo "Usage: $0 [${supported_versions}]"
-        exit 1
-    fi
-}
-
 function check_os() {
-    os=$1
+	os=$1
```

Comment:
Seems to be lots of unrelated whitespace changes..

Reply:
Will check and update the unrelated spaces.